package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "lint": "eslint src/",
     "mocha": "mocha --compilers js:babel/register,scss:power-ui/test-utils/mocha-scss-compiler.js src/**/test.js",
+    "cover": "babel-node ./node_modules/.bin/isparta cover ./node_modules/.bin/_mocha -- --compilers scss:power-ui/test-utils/mocha-scss-compiler.js src/**/test.js",
```

Comment:
mocha is just a wrapper for _mocha. It spawns a new process that runs the _mocha. AFAIK The way isparta handles the transformations to es5 is not compatible with the wrapper. This way the coverage reports are original, untransformed, format.